remote-facade/README.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: pattern
+title: Remote Facade Pattern
+folder: remote-facade
+permalink: /patterns/remote-facade/
+categories: Architectural
+language: en
+tags:
+- Performance
+---
```

Comment:
Please check the yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute